externals/CMakeLists.txt

```diff
@@ -142,6 +142,12 @@ if (WIN32)
     target_compile_options(sirit PUBLIC "-Wno-error=unused-command-line-argument")
 endif()
 
+# half
+if (APPLE AND NOT TARGET half)
```

Comment:
The non-imported target `half` will never exist at this point you know :p

Reply:
There's no reason not to check in case it may be imported externally at some point.

Reply:
The imported name will probably be something like `half::half`, if it ever exists.